android/app/build.gradle

```diff
@@ -255,6 +255,7 @@ android {
 }
 
 dependencies {
+    implementation project(':@react-native-community_netinfo')
```

Comment:
A question - so there was a problem with resolving scoped module paths that requires this underscore symbol replacement?

Reply:
A question - why does it have "@" ?

Reply:
it's part of the package name, isn't it? This was added by `react native link`

Reply:
> A question - so there was a problem with resolving scoped module paths that requires this underscore symbol replacement?

I imagine this is a Maven/Gradle limitation, so while the node package uses `/`, it needs to be replaced with an underscore in the Gradle files.